handler/src/main/java/io/netty/handler/ssl/ReferenceCountedOpenSslContext.java

```diff
@@ -342,6 +342,9 @@ public String run() {
             if (enableOcsp) {
                 SSLContext.enableOcsp(ctx, isClient());
             }
+
+            // TODO: Make the configurable.
```

Comment:
default on / off?

Reply:
I think I will make if off for now and allow to configure it with a System property... WDYT @ejona86 @johnou . Alternative I could allow to configure it via the `OpenSslContext`

Reply:
Is it just that we want to avoid unstable code, or is there a reason to believe it would have a performance cost?

Reply:
@ejona86 its more to allow to test it a bit more before make it default. I also still need to do some performance tests but I suspect the impact should be almost zero.

Reply:
Then system property sounds good. When we are comfortable we change the default. When it is "proven" we remove the option.

Reply:
Alright I introduced the system property "io.netty.handler.ssl.openssl.useTasks"